documents/sphinx-custom.css

```diff
@@ -0,0 +1,207 @@
+/*
+Dark Mode colors and other variables copied from "doxygen-awesome.css".
+We could potentially import the entire style sheet here, but we're only
+interested in the color definitions and a select few other styles.
+DoxygenAwesome is licensed under the MIT License.
+*/
+html {
+    /* font-families. will affect all text on the website
+     * font-family: the normal font for text, headlines, menus
+     * font-family-monospace: used for preformatted text in memtitle, code, fragments
+     */
+    --font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;
+    --font-family-monospace: ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace;
+
+    /* font sizes */
+    --page-font-size: 15.6px;
+    --navigation-font-size: 14.4px;
+    --toc-font-size: 13.4px;
+    --code-font-size: 14px; /* affects code, fragment */
+    --title-font-size: 22px;
+
+    /* color for all separators on the website: hr, borders, ... */
+    --separator-color: #dedede;
+
+    /* border radius for all rounded components. Will affect many components, like dropdowns, memitems, codeblocks, ... */
+    --border-radius-large: 8px;
+    --border-radius-small: 4px;
+    --border-radius-medium: 6px;
+
+    color-scheme: dark;
+
+    --primary-color: #1982d2;
+    --primary-dark-color: #86a9c4;
+    --primary-light-color: #4779ac;
+
+    --box-shadow: 0 2px 8px 0 rgba(0,0,0,.30);
```

Comment:
Doesn't need to be done with this PR, but as "accessibility" is a active area in ASWF, perhaps it's worthwhile to run an accessibility test on the page for the colors and fonts ?

Reply:
Oh, good point!

Given that we're aiming for some consistency in styles between the various MaterialX sites and docs, could we maybe make this a separate ticket, to look at accessibility across all of the docs?